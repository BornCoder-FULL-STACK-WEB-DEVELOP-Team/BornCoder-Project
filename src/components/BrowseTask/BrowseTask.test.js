import React from "react";
import ReactDOM from "react-dom";
import BroesTask from "./BrowseTask";
import { render } from "@testing-library/react";
import axios from "axios";

it("renders welcome message", () => {
  const { getByText } = render(<BroesTask />);
  expect(getByText("open")).toBeInTheDocument();
  expect(getByText("open")).toMatchSnapshot();
});

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<BroesTask />, div);
});
