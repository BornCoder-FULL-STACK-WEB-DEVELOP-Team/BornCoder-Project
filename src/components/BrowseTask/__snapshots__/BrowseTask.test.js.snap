// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders welcome message 1`] = `
<span
  class="step-button button-active"
>
  open
</span>
`;
